app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.js

```diff
@@ -62,13 +62,13 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 
   Ga4LinkTracker.prototype.trackClick = function (event) {
     var element = event.target
-
+    var trackFunctions = window.GOVUK.analyticsGa4.core.trackFunctions
```

Comment:
I'm in two minds about this. It does make the code cleaner and easier to read, but at the same time it slightly obfuscates what is being called by this variable - you have to read this bit of the code to know what `trackFunctions` refers to (although it is named the same thing, so that's good).

It might make a difference to the asset size on the public site, but only fractionally, and it's possible any saving would be negated by gzip/minifying anyway.

In conclusion... I don't know! It's probably fine 😁

Reply:
Thanks @andysellick - I've asked another dev and they liked the refactor, so will keep it for now :+1: